osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs

```diff
@@ -171,16 +142,5 @@ protected override void Dispose(bool isDisposing)
                 entry.UnbindEvents();
             lifetimeEntries.Clear();
         }
-
-        private class DrawablePoolNoLifetime<T> : DrawablePool<T>
```

Comment:
Not seeing an explanation why this has been removed as well.

Reply:
`RemoveWhenNotAlive` is not doing anything for a `DrawablePoll` and I think the original implementation was just mixing something up (pooled drawables are not `InternalChildren` of `DrawablePool`, and not checked for child lifetime).
I thought it is due to changes of `DrawablePool` making it unnecessary, but commit history is contradicting that hypothesis.

Reply:
This existed because `LifetimeManagementContainer` doesn't support `RemoveWhenNotAlive => true`. But this hasn't been using LMC for a long time.